### PR TITLE
Translated DE-ENG sessions.mdx

### PR DESCRIPTION
**Pull request by Gertjezzz**
Translated German to English

### CODE REVIEW

**Comment by Gertjezzz** [comment]
Is 'PV' in this file meant to be 'solar'?

**Comment by Gertjezzz** [comment]
Build failed, dont know on what - a broken link maybe?

File: i18n/en/docusaurus-plugin-content-docs/current/features/sessions.mdx [hunk 1]
```diff
@@ -6,94 +6,89 @@ import Screenshot from "../../../../../src/components/Screenshot";
...
+        Alternatively, the remaining time is calculated based on the current charging power and the charging target.
 
-        * **Ladedauer:** Zeigt die effektive Dauer des Ladevorgangs an.
-        Wird der Ladevorgang pausiert (bspw. aufgrund von PV-Überschuss), wird die Zeit angehalten.
+        * **Charging duration:** Shows the effective duration of the charging session.
+        If the charging session is paused (e.g. due to PV surplus), the time is stopped.
 
-        * **Sonne:** Der Prozentwert zeigt, welcher Anteil der geladenen Energie aus der PV-Anlage stammt.
-        Energie aus dem Hausspeicher wird als Sonnenenergie gewertet.
+        * **Sun:** The percentage value shows which proportion of the charged energy comes from the PV system.
```
Comment (naltatis) [inline i18n/en/docusaurus-plugin-content-docs/current/features/sessions.mdx:41]: Bildschirmfoto 2024-09-15 um 08 29 03

Das heißt in diesem Kontext "solar". Du kannst viele der Wordings in der Demo-Instanz https://demo.evcc.io/ sehen wenn du auf Englisch stellst.

**Comment by naltatis** [comment]
> Build failed, dont know on what - a broken link maybe?

Not your fault. There was an issue in the main branch. I'll fix this, change the wording and merge.

Thanks for your work 🙌

**Pull request merged**